content/the-ember-times-issue-186.md

```diff
@@ -52,15 +35,19 @@ Read more about the different strategies to avoiding lifecycle in components at
 
 ---
 
-## [4. Section title in sentence case 🐹](section-url)
+## [`ember-responsive-image` v3 release 🖼](https://github.com/kaliber5/ember-responsive-image)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Optimising image size and formats to improve performance can get complicated, but the [`ember-responsive-image`](https://github.com/kaliber5/ember-responsive-image) addon has your back! Now you have even more reason to take advantage of its abilities following the annoucement by [Simon Ihmig (@simonihmig)](https://github.com/simonihmig) of [version 3](https://github.com/kaliber5/ember-responsive-image/releases/tag/v3.0.0). This release provides support for Cloudinary and imgix image CDNs, further enriching a full set of functionality for re-sizing images and optimizing image loading times across all browsers. Go check it out.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+---
+
+## [Announcing Glimmer Apollo 🎉](https://glimmer-apollo.com/)
+
+[Jose Marluedke (@josemarluedke)](https://github.com/josemarluedke) shared [Glimmer Apollo](https://glimmer-apollo.com/) on [Twitter](https://twitter.com/josemarluedke/status/1410295473072611328), a new addon that integrates GraphQL in your Ember and Glimmer apps with a declarative API to query, mutate, and access GraphQL data.
```

Comment:
@josemarluedke if you have anything you want added / edited, please let me know! hoping to get the issue out tomorrow in the PM pacific time (got delayed due to being under the weather). it sounds really cool just not sure if i 100% follow everything yet

btw, the docs site looks great, it's docfy?

Reply:
Just a correction to my name. It's Josemar Luedke.

```suggestion
[Josemar Luedke (@josemarluedke)](https://github.com/josemarluedke) shared [Glimmer Apollo](https://glimmer-apollo.com/) on [Twitter](https://twitter.com/josemarluedke/status/1410295473072611328), a new addon that integrates GraphQL in your Ember and Glimmer apps with a declarative API to query, mutate, and access GraphQL data.
```

Reply:
sorry, bad copy pasta on my part! 🤦‍♀️

Reply:
> sorry, bad copy pasta on my part! 🤦‍♀️

No worries.

> btw, the docs site looks great, it's docfy?

Thanks. Yes, it's Docfy. The UI is built on a theme (ember addon) that is intended to be shared and usable with other docfy apps, I just need to finish that up. https://github.com/josemarluedke/docfy-theme-ember